InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -127,4 +132,23 @@ public void rotateToDeg(double targetPoint) {
         targetChange = 0;
     }
 
-}
\ No newline at end of file
+    public void rotateByDeg(double degrees) {
+        rotateToDeg(getTargetTurretDegGivenOffset(degrees));

```

Comment:
see the comment under rotateTurretGivenLLOffset. while we don't need to use rotateByDeg, it might be a good idea to keep it just in case. if we do keep it, there is one thing to keep in mind: getTargetTurretDegGivenOffSet was written for the Limelight, so it has the convention that a negative value will increase the angle. would this be the functionality we want?